core/src/services/webhdfs/writer.rs

```diff
@@ -59,11 +97,59 @@ impl oio::OneShotWrite for WebhdfsWriter {
         match status {
             StatusCode::CREATED | StatusCode::OK => {
                 resp.into_body().consume().await?;
+
                 Ok(())
             }
             _ => Err(parse_error(resp).await?),
         }
     }
+
+    async fn complete_block(&self, block_ids: Vec<String>) -> Result<()> {
+        // create a empty file to concat blocks
```

Comment:
Why we need this?

Reply:
```shell
curl -i -X POST "http://<HOST>:<PORT>/webhdfs/v1/<PATH>?op=CONCAT&sources=<PATHS>"
```
CONCAT op will concat sources into path, in `write_block` we create sources, so in `complete_block` we should create a empty path concat sources

Reply:
> CONCAT op will concat sources into path, in `write_block` we create sources, so in `complete_block` we should create a empty path concat sources

The source can't be a non-exist path?

Reply:
Both path and sources must exist.

Reply:
Got it.